### PR TITLE
Allow HttpConfiguration to Jetty server

### PR DESCRIPTION
**Pull request by ohadza**

### CODE REVIEW

File: src/main/java/io/logz/guice/jersey/JerseyServer.java [hunk 1]
```diff
@@ -53,7 +54,7 @@ public void stop() throws Exception {
     private void configureServer() {
         List<ServerConnectorConfiguration> serverConnectorConfigurations = jerseyConfiguration.getServerConnectors();
         serverConnectorConfigurations.forEach(configuration -> {
-            ServerConnector connector = new ServerConnector(server);
+            ServerConnector connector = new ServerConnector(server, new HttpConnectionFactory(configuration.getHttpConfiguration()));
```
Comment (asafm) [inline src/main/java/io/logz/guice/jersey/JerseyServer.java:57]: let's do this like the rest:
```
connector.setConnectionFactories(new HttpConnectionFactory(configuration.getHttpConfiguration()))
```

File: src/main/java/io/logz/guice/jersey/configuration/JerseyConfigurationBuilder.java [hunk 2]
```diff
@@ -38,6 +39,11 @@ public JerseyConfigurationBuilder addPort(int port) {
         return this;
     }
 
+    public JerseyConfigurationBuilder addPortWithHttpConfiguration(int port, HttpConfiguration httpConfiguration) {
```
Comment (asafm) [inline src/main/java/io/logz/guice/jersey/configuration/JerseyConfigurationBuilder.java:42]: It's too many flavors. Let's create a `ServerConnectorConfigurationBuilder` and do this right

File: src/test/java/io/logz/guice/jersey/supplier/JerseyServerSupplier.java [hunk 3]
```diff
@@ -38,7 +39,7 @@ public static void createServerAndTest(JerseyConfigurationBuilder configurationB
                                            JettyServerCreator jettyServerCreator,
                                            Tester tester) throws Exception {
         int port = FreePortFinder.findFreeLocalPort();
-        configurationBuilder.addPort(port);
+        configurationBuilder.addPortWithHttpConfiguration(port, new HttpConfiguration());
```
Comment (asafm) [inline src/test/java/io/logz/guice/jersey/supplier/JerseyServerSupplier.java:42]: Let's add a small test that shows I configure jetty with it and it works. Max request header sounds about right

**Review by asafalima — APPROVED** [review]

**Review by asafm — CHANGES_REQUESTED** [review]
Let's also create a release - change version to 0.13

**Review by asafm — APPROVED** [review]

**Pull request merged**